perfkitbenchmarker/configs/benchmark_config_spec.py

```diff
@@ -380,6 +380,12 @@ def _GetOptionDecoderConstructions(cls):
         'high_availability': (option_decoders.BooleanDecoder, {
             'default': False
         }),
+        'backup_enabled': (option_decoders.BooleanDecoder, {
```

Comment:
im confused... this is in the database_spec, and is only applicable inside said spec. like
```
managed_relational_database:
  cloud: GCP
  database_name: foo
  backup_enabled: False
```